packages/amplify-graphql-schema-generator/src/__tests__/__snapshots__/generate.test.ts.snap

```diff
@@ -145,6 +145,19 @@ type Story @refersTo(name: \\"stories\\") @model {
 "
 `;
 
+exports[`generate graphqlSchemaFromSQLSchema creates graphql schema from "postgres" "serial" schema 1`] = `
```

Comment:
This snapshot is not intentional as we are not intending to support GQL generation here. It is generated automatically to test against anyway from the schemas in `__utils__`.